src/frontend/apps/standalone_site/src/features/ClassRooms/components/ClassRooms.tsx

```diff
@@ -0,0 +1,133 @@
+import styled from 'styled-components';
+
+import { ReactComponent as ClassroomsIcon } from 'assets/svg/iko_webinairesvg.svg';
+import { WhiteCard, ContentCards } from 'components/Cards';
+import { ContentSpinner } from 'components/Spinner';
+
+import ClassRoom from './ClassRoom';
+
+const messages = defineMessages({
+  ClassroomTitle: {
+    defaultMessage: 'Classrooms',
+    description: 'Classrooms title',
+    id: 'features.ClassRooms.ClassroomTitle',
+  },
+  NoClassroom: {
+    defaultMessage: 'There is no classroom to display.',
+    description: 'Text when there is no classroom to display.',
+    id: 'features.ClassRooms.NoClassroom',
+  },
+  Error: {
+    defaultMessage: 'Sorry, an error has occurred.',
+    description: 'Text when there is an error.',
+    id: 'features.ClassRooms.Error',
+  },
+});
+
+const BoxPagination = styled(Box)`
+  & button {
+    color: #055fd2;
+    background-color: #f2f7fd;
+  }
+  & button[aria-current] {
+    background-color: #dbebff;
+  }
+  & button svg {
+    stroke: #055fd2;
+  }
+`;
+
+const ContainerInfo = ({ children }: PropsWithChildren<ReactNode>) => {
+  return (
+    <Box
+      direction="row"
+      align="center"
+      justify="center"
+      margin={{ top: 'medium' }}
+      gap="small"
+    >
+      {children}
+    </Box>
+  );
+};
+
+function ClassRooms() {
+  const intl = useIntl();
+  const nbItemPerPage = 20;
+  const [currentPage, setCurrentPage] = useState(0);
+
+  const {
+    isLoading,
+    isError,
+    data: classRooms,
+  } = useClassrooms(
+    {
+      limit: String(nbItemPerPage),
+      offset: String(currentPage * nbItemPerPage),
+    },
+    { keepPreviousData: true, staleTime: 20000 },
+  );
+
+  let content = (
+    <ContainerInfo>
+      <ClassroomsIcon width={80} height={80} />
+      <Text weight="bold">{intl.formatMessage(messages.NoClassroom)}</Text>
+    </ContainerInfo>
+  );
+
+  if (isError) {
+    content = (
+      <ContainerInfo>
+        <Alert size="large" color="#df8c00" />
+        <Text weight="bold">{intl.formatMessage(messages.Error)}</Text>
+      </ContainerInfo>
+    );
+  } else if (isLoading) {
+    content = <ContentSpinner />;
+  } else if (classRooms?.results.length) {
+    content = (
+      <Fragment>
+        <ContentCards margin={{ top: 'medium' }}>
+          {classRooms.results.map((classroom) => (
+            <ClassRoom
+              key={`classroom-${classroom.id}`}
+              classroom={classroom}
+            />
+          ))}
+        </ContentCards>
+        {classRooms.count > nbItemPerPage && (
+          <BoxPagination>
+            <Pagination
+              numberItems={classRooms.count}
+              onChange={({ page: newPage }: { page: number }) => {
+                setCurrentPage(newPage - 1);
+              }}
+              page={currentPage + 1}
+              step={nbItemPerPage}
+              alignSelf="center"
+              margin={{ top: 'medium' }}
+            />
+          </BoxPagination>
```

Comment:
For this we have to be consistent. On the playlist list, no matter if there is one page or many, the pagination is always shown. WDYT @RVassili